openslides/__main__.py

```diff
@@ -104,9 +104,9 @@ def get_parser():
         help='The used settings file. The file is created, if it does not exist.')
     subcommand_start.set_defaults(callback=start)
     subcommand_start.add_argument(
-        '--development',
+        '--local-installation',
         action='store_true',
-        help='Option for development purposes.')
+        help='Store settings and user files in a local directory.')
```

Comment:
I think this option is only for *create* the directory (by strating openslides on first time) - not for further start commands. Suggestionn for help text: `Create database and settings file in a local subdirectory of this current directory.`